### DISCUSSION TITLE
iframes and embedded HTML

### DISCUSSION THREAD

**Discussion by augustfly** [knowledgefutures/pubpub#3007] [Ideas]
Tools like [x3d](https://www.web3d.org/x3d/stylesheets/python/python.html), [matplotlib+D3](mpld3), [bokeh](https://bokeh.org/) and [plotly](https://plotly.com/python/) can be used to build and export stand-alone interactive HTML figures. Can these be directly supported in the UI? 

I managed a work around, which was to upload the HTML as an "other" asset, which renders it as a link in the article. Following that link gives me a new URL on assets.pubpub.org.

Then I used the resulting assets.pubpub.org link and added the figure as an iFrame. But this seems like a kludge! 

Also +100 for making the iFrame's overflow the text by some amount. [This PubPub example](https://help.pubpub.org/pub/interactive-articles/release/5) suggests it is possible but not in my UI (I only have the option of a scroll bar).

**Comment by meeloun** [comment]
When choosing essay ghostwriting         [http://www.pnstudy.com](http://www.pnstudy.com)         services for international students, it is important to pay attention to multiple aspects to ensure that your choice is correct. By carefully identifying and communicating with institutions, it can help you find personalized services that truly meet your needs.
